### PR TITLE
Check NodeJS version and log FF version

### LINKED ISSUE #241: Add explicit check for compatible Node version on startup
### Description

Whilst we document Node 16 as our required version, we don't check it when we start.... which means users could unintentionally run with an older version and hit unexpected issues.

We should have an explicit check on startup and refuse to start if it isn't 16+

### Epic/Story

_No response_

### PR DESCRIPTION
**Pull request by hardillb**
Fixes #241 and #242

### CODE REVIEW

**Comment by hardillb** [comment]
Should we hard exit or just log NodeJS version?

File: forge/config/index.js [hunk 1]
```diff
@@ -35,6 +36,21 @@ module.exports = fp(async function(app, _opts, next) {
...
+        //npm start
+        app.log.info(`FlowForge v${process.env.npm_package_version}`)
+    } else {
+        //everything else
+        let {version} = require(path.join(module.parent.path, "..", "package.json"))
+        app.log.info(`FlowForge v${version}`)
+    }
+
+    if (!semver.satisfies(process.version,">=16.0.0")) {
+        app.log.warn(`FlowForge requires at least NodeJS v16, ${process.version} found`)
```
Comment (knolleary) [inline forge/config/index.js:49]: I'd be tempted to move this logic right to start of the main app.js, before we try doing any setup - and console.error and exit if node isn't suitable.

**Comment by hardillb** [comment]
The original task #241 says to exit so have added that.

File: forge/config/index.js (hunk 1 above)
Comment (hardillb) [reply forge/config/index.js:49]: OK, makes sense. I was looking for the first place I could get hold of a `app.log` object. We can still log it here so it's clean for a good version

**Pull request merged**